test_scripts/redhat/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM registry.access.redhat.com/ubi8/ubi:8.1
+FROM registry.access.redhat.com/ubi8/ubi:8.3
 
 
 COPY . /app
+RUN yum -y install initscripts procps which hostname && yum clean all
```

Comment:
I think you always need an update, otherwise you can have stale metadata for your repo (or none if the base is set up right).

Reply:
👍